org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/NarrativeViewer.java

```diff
@@ -286,11 +286,19 @@ public void setDTG(final HiResDate dtg)
    */
   public void setEntry(final NarrativeEntry entry)
   {
-    // select this item
-    viewer.setSelection(new StructuredSelection(entry));
-
-    // make sure the new item is visible
-    viewer.reveal(entry);
+    try
+    {
+      viewer.getGrid().setRedraw(false);

```

Comment:
@IanMayo  by doing this it stop all rendering of ui so live animation of scroll not updated so soon after we do all processing we then enable update ui so it just need to update on it view port.